static/sass/_pattern_p-strip.scss

```diff
@@ -299,6 +299,13 @@
     }
   }
 
+  .p-strip--dark-blue {
```

Comment:
Why not use `p-strip--accent` which is what this is for?

Reply:
to be honest i think we have that `$color-accent` already set to a different colour. This website has so many accent colours... i'll try to see if I can use your suggestions.